lms/static/scripts/frontend_apps/utils/grader-service.js

```diff
@@ -1,13 +1,31 @@
 import { apiCall } from './api';
 
 /**
- * Submits the grade to the LTI endpoint, waits for it to return, then updates the state with the
- * new grade
+ * @typedef {Object} Student
+ * @prop {string} LISResultSourcedId - The unique student id
+ * @prop {string} LISOutcomeServiceUrl - The student's service url
```

Comment:
I'd describe this parameter as "API URL for posting outcome results"

Reply:
will do!